ui/job-view/details/tabs/PerformanceTab.jsx

```diff
@@ -16,6 +86,63 @@ export default class PerformanceTab extends React.PureComponent {
         role="region"
         aria-label="Performance"
       >
+        <div className="performance-panel-actions">
+          {
+            // If there is a profiler link, show this first. This is most likely
+            // the primary action of the user here.
+            profilerLink
+          }
+          {
+            // Just to be safe, use the same isPerfTest check the other
+            // "Create Gecko Profile" button uses in the action menu.
+            isPerfTest(selectedJobFull) ? (
+              <button
+                type="button"
+                className={`btn btn-${
+                  // Only make this primary if there is no profiler link.
+                  profilerLink ? 'secondary' : 'primary'
+                } btn-sm`}
+                onClick={this.createGeckoProfile}
+                title={
+                  'Trigger another run of this test with the profiler enabled. The ' +
+                  'profile can then be viewed in the Firefox Profiler.'
+                }
+              >
+                <FontAwesomeIcon icon={faRedo} className="mr-2" />
+                {profilerLink
+                  ? 'Re-trigger performance profile'
+                  : 'Generate performance profile'}
+              </button>
+            ) : null
+          }
+          <a
+            href={getCompareChooserUrl({
+              newProject: repoName,
+              newRevision: revision,
+            })}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="btn btn-secondary btn-sm"
+          >
+            <FontAwesomeIcon icon={faTable} className="mr-2" />
+            Compare against another revision
+          </a>
+        </div>
+        {
+          // It can be confusing after triggering a profile what happens next. The
+          // job list only gets populated later. This notification will help the
+          // user know the next action.
+          triggeredGeckoProfiles > 0 ? (
+            <div className="alert alert-info m-1" role="alert">
```

Comment:
Please use the bootstrap `Alert` component. I think there's still some old code that's using the bootstrap styles in this manner but we've moved away from that in favor of the components.

Reply:
Ah nice, I didn't know about all of these components. I'll check it out and migrate it.